docs/src/content/hardhat-runner/docs/advanced/scripts.md

```diff
@@ -46,37 +46,29 @@ drwxr-xr-x    3 fzeoli  staff      96 Jul 30 15:27 scripts
 drwxr-xr-x    3 fzeoli  staff      96 Jul 30 15:27 test
 ```
 
-Inside `scripts/` you will find `sample-script.js`. Read through its comments to have a better idea of what it does.
+Inside `scripts/` you will find `deploy.js`. Read through its comments to have a better idea of what it does.
 
-TODO @/../packages/hardhat-core/sample-projects/basic/scripts/sample-script.js
+<<< @/../packages/hardhat-core/sample-projects/javascript/scripts/deploy.js
 
-Done? Before running the script with `node` you need to declare `ethers`. This is needed because Hardhat won't be injecting it on the global scope as it does when calling the `run` task.
+Now run the script:
 
-```js{2}
-const hre = require("hardhat");
-const ethers = hre.ethers;
-
-async function main() {
-  //...
-}
 ```
-
-Now you're ready to run the script:
-
-```
-$ node scripts/sample-script.js
-Greeter address: 0x5FbDB2315678afecb367f032d93F642f64180aa3
+$ node scripts/deploy.js
+Lock with 1 ETH deployed to: 0x5FbDB2315678afecb367f032d93F642f64180aa3
 ```
 
-By accessing the [Hardhat Runtime Environment] at the top, you are allowed to run the script in a standalone fashion. Hardhat always runs the compile task when it's invoked via `npx hardhat run`, but in a standalone fashion you may want to call compile manually to make sure everything is compiled. This is done by calling `hre.run('compile')`. Uncomment the following line and re-run the script with `node`:
+By accessing the [Hardhat Runtime Environment](../advanced/hardhat-runtime-environment.md) at the top, you are allowed to run the script in a standalone fashion.
+
+Hardhat always runs the compile task when it's invoked via `npx hardhat run`, but in a standalone fashion you may want to call compile manually to make sure everything is compiled. This is done by calling `hre.run("compile")`. Add the following line at the beginning of the `main` function and re-run the script with node:
```

Comment:
```suggestion
Hardhat always runs the compile task when it's invoked via `npx hardhat run`, but in a standalone fashion you may want to call compile manually to make sure everything is compiled. This can be done by calling `hre.run("compile")`. Add the following line at the beginning of the `main` function and re-run the script with node:
```